subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties

```diff
@@ -1,7 +1,7 @@
 #Generated file, please do not edit - Version values used in build-init templates
 commons-math=3.6.1
 commons-text=1.9
-groovy=3.0.10
+groovy=3.0.11
```

Comment:
I think the Groovy version for the init plugin should already be Groovy 4.x. Is there any reason to keep it on 3.x? The version here shouldn't have anything to do with the version we ship in Gradle.